### ISSUE TITLE
Webpage to mark the route

### ISSUE THREAD

**Issue by diegolhambi** [SubstituteCS/Pokesposed#14]
And if the app provide a link for you to do the route on your computer? Without all the change to pokemon go, change to pokesposed every time

A local web server running in some port, not need any cloud or a server, like the airdroid do

**Comment by SubstituteR** [comment]
Do you mean run a web-server that communicates the same way the application does?

**Comment by diegolhambi** [comment]
Yeah, a web-server with a page to mark the route in a large screen.
